plugins/obs-webrtc/whip-output.cpp

```diff
@@ -23,6 +23,7 @@ WHIPOutput::WHIPOutput(obs_data_t *, obs_output_t *output)
 	  running(false),
 	  start_stop_mutex(),
 	  start_stop_thread(),
+	  base_ssrc(generate_random_u32()),
```

Comment:
As I'm not familiar with WebRTC, can you explain to me how this change is related to user agents and `OPTIONS` requests?

While the PR has been dutifully squashed into a single commit, its message will not suggest to me that `ssrc` generation was also changed as part of it. I'm not opposed to merging it as part of this PR, but I'd rather see those changes moved into its own commit (so _if_ issues arise with it, we don't have to manually pull the changes apart to revert it).